frontend/src/domains/places/components/SearchList/SearchList.tsx

```diff
@@ -32,7 +33,18 @@ const SearchList = ({ searchResults, handleSelect }: SearchListProps) => {
                 width="100%"
               >
                 <Text variant="caption">{searchResult.name}</Text>
-                <Text variant="label">{searchResult.roadAddressName}</Text>
+
+                {searchResult.roadAddressName ? (
+                  <SearchAddress
+                    addressType="도로명"
+                    address={searchResult.roadAddressName}
+                  />
+                ) : (
+                  <SearchAddress
+                    addressType="지번"
+                    address={searchResult.addressName}
+                  />
+                )}
```

Comment:
여기를 이렇게 사용할 수 있을 것 같은데 어떻게 생각하시나요??

```suggestion

                <SearchAddress
                addressType={addressType}
                address={address}
              />
```